src/TouristInfoCards.js

```diff
@@ -0,0 +1,49 @@
+import React from "react";
+
+const TouristInfoCards = () => {
+  return (
+    <div className="card-container">
+      <Card
+        link="https://peoplemakeglasgow.com"
+        image="https://media.cntraveler.com/photos/5ad4ece4379a3a44feb968b6/master/w_3604,h_2574,c_limit/Colosseum-GettyImages-537714366.jpg"
+        name="Glasgow"
+        text="Youe guaranteed to find accommodation in Glasgow "
+      />
+
+      <Card
+        link="https://visitmanchester.com"
+        image="https://thumbs.dreamstime.com/b/manchester-canal-uk-england-skyline-shot-deansgate-108118828.jpg"
+        name="Manchester"
+        text="Manchester city centre will be getting into the Christmas spirit with its series of Festive Sundays"
+      />
+
+      <Card
+        link="https://visitlondon.com"
+        image="https://media.istockphoto.com/id/1294454411/photo/london-symbols-with-big-ben-double-decker-buses-and-red-phone-booth-in-england-uk.jpg?b=1&s=170667a&w=0&k=20&c=SR5FBt8VvpZEOIxZcTmoEf9XxU0oiSnbobD3E6ZB358="
+        name="London"
+        text="Welcome to London! Discover the best of London with Visit London."
+      />
+    </div>
+  );
+};
+
+const Card = props => {
```

Comment:
using a reusable `Card` component here is a really good idea, as you're essentially going to be using the same card layout for each city. I would pull this out into it's own file, rather than having it at the bottom here (and import it at the top). Having it in it's own file means it can be used throughout the app if needed and not just in `TouristInfoCards`.

You could also get rid of some repetition in the top section - for example if you have an array of data for all the cities, you could map over this, rendering a `Card` component for each. Imagine if you wanted to add 10 more city cards, your file would soon get pretty huge, so using a .map would help with this